Artifacts/windows-ConfigurationManagerCurrentBranch(EVAL)/setupCMCB.ps1

```diff
@@ -86,15 +53,20 @@ if((gwmi win32_computersystem).partofdomain -eq $true)
     Stop-Service 'MSSQLSERVER' -Force
     Start-Service 'MSSQLSERVER'
 
+	#Disable defender to reduce load during setup	
+	try { Set-MpPreference -DisableRealtimeMonitoring $true } catch{}
```

Comment:
Presumably this should be turned back on at the end of a script (in a finally block). I don't think people would expect this change as a byproduct of running an artifact.